### PR TITLE
footer completed

### PR DESCRIPTION
**Pull request by okorie2**
footer created for the webapp

### CODE REVIEW

**Review by highman95 — APPROVED** [review]

**Review by SaidiAli — COMMENTED** [review]
Resolve conflict please. Will merge after.

File: src/App.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
 import React,{Component} from 'react';
```
Comment (Austinove) [inline src/App.js:2]: Please @okorie2 I think you try to use either a function of the class but not with both named with the same name.
Try to combine them and check if the whole application is running successfully.

File: src/App.js [hunk 2]
```diff
@@ -9,6 +10,22 @@ class App extends Component{
       </div>
     )
   }
```
Comment (Austinove) [inline src/App.js:12]: @okorie2  I think you should add the footer in the `index.js` in the `components ` folder where `AppHeader `is also linked

**Review by Austinove — CHANGES_REQUESTED** [review]
Please look at the comments i have put in your changes. this will make the whole flow of the project. dont forget to pull updates from `staging-1` before pushing

**Comment by okorie2** [comment]
In a bid to properly resolve conflicts, I'm closing this pr. A new one will be made

**Pull request closed without merging**